Frameworks/SafariServices/SFSafariWebViewController.mm

```diff
@@ -49,13 +70,24 @@ - (BOOL)webView:(UIWebView*)webView
     shouldStartLoadWithRequest:(NSURLRequest*)request
                 navigationType:(UIWebViewNavigationType)navigationType {
     NSURL* url = request.URL;
+    BOOL openWithAppDelegate = NO;
 
     // Check whether this request is for the redirect URL that we're watching out for
     // (modulo the query string and fragment, which are allowed to vary)
-
     if ([url.scheme isEqualToString:_redirectUrl.scheme] && [url.host isEqualToString:_redirectUrl.host] &&
```

Comment:
I could, but I feel the code would be less clear that way.

---
In reply to: [83669085](https://github.com/Microsoft/WinObjC/pull/1147#discussion_r83669085) [](ancestors = 83669085)